### DISCUSSION TITLE
OR TOOL CVRP - Get traffic data in OSRM

### DISCUSSION THREAD

**Discussion by Shivara2022** [google/or-tools#3729] [Q&A]
Hi All,

I'm using OSRM map dataset to build a distance matrix. But the travel time between nodes was almost 5/10 mins delay for a single node when compared google maps.

Is there a way to closer travel time in OSRM?

**Comment by jmarca** [comment]
That’s an OSRM question, not an ORTools question. JamesOn Apr 3, 2023, at 10:13, Shivara2022 ***@***.***> wrote:﻿
Hi All,
I'm using OSRM map dataset to build a distance matrix. But the travel time between nodes was almost 5/10 mins delay for a single node when compared google maps.
Is there a way to closer travel time in OSRM?

**Comment by jmarca** [comment]
But aside from the fact that you should ask the OSRM project that question, I’d recommend just scaling up the OSRM (or Valhalla) travel times by a factor, say 1.1. Or round *UP* to the nearest 5 or even 15 minutes. Given the inherent variability of travel times in congested areas, trying for more accuracy is a fool’s errand. JamesOn Apr 3, 2023, at 10:13, Shivara2022 ***@***.***> wrote:﻿
Hi All,
I'm using OSRM map dataset to build a distance matrix. But the travel time between nodes was almost 5/10 mins delay for a single node when compared google maps.
Is there a way to closer travel time in OSRM?
